config/typo3.rules

```diff
@@ -0,0 +1,69 @@
+# Harden the `chmod` function
+sp.disable_function.function("chmod").param("mode").filename_r("typo3/sysext/core/Classes/Utility/GeneralUtility.php$").value_r("^[0-9]{2}6$").allow();
+sp.disable_function.function("chmod").param("mode").value_r("^[0-9]{2}[67]$").drop();
+sp.disable_function.function("chmod").param("mode").value_r("o\\+w$").drop();
```

Comment:
This doesn't work, since the parameter `mode` is [expecting an integer]( https://secure.php.net/manual/en/function.chmod.php ).